certified-connectors/Rainbird/apiProperties.json

```diff
@@ -0,0 +1,53 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Api key",
+          "description": "The api key associated to your Rainbird account",
+          "tooltip": "Provide the Api key, you can find it in your Rainbird account details",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "optional password",
+          "description": "No password required, type any character",
+          "tooltip": "No password required, type any character",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#2E1648",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Dynamic host",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@queryParameters('environment')",
+          "x-ms-apimTemplate-operationName": [
+            "Start",
+            "Inject",
+            "Query",
+            "Response",
+            "Undo",
+            "Evidence",
+            "Version"
+          ]
+        }
+      }
+    ],
+    "publisher": "Mathias Rigaud"
```

Comment:
@Mat-Rig publisher property should be a name of company or organization. The stackOwner property is missing, please refer to the [apiProperties.json template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json) for more information about it.